apps/studio/src/pages/home/library/MintSong.tsx

```diff
@@ -302,206 +212,81 @@ const MintSong = () => {
-                    <Typography variant="subtitle1">
-                      You&apos;re almost ready to mint! To proceed please review
-                      your ownership contract and follow the steps below.
-                    </Typography>
-                  </Stack>
-
-                  <ConfirmContract
-                    isCoCreator={ values.owners.length > 1 }
-                    songTitle={ title }
-                    onConfirm={ (value: boolean) =>
-                      setFieldValue("consentsToContract", value)
+            <Stack pt={ 3 }>
+              <Stack spacing={ 5 }>
+                <Box>
+                  <SwitchInputField
+                    description={
+                      "Minting a song will create an NFT that reflects " +
+                      "ownership, making streaming royalties purchasable. " +
+                      "Once a song is minted, it cannot be deleted."
                     }
+                    disabled={ true }
+                    includeBorder={ false }
+                    name="isMinting"
+                    title="DISTRIBUTE & MINT SONG"
                   />
 
-                  <HorizontalLine sx={ { my: 5 } } />
+                  <SelectCoCreators
+                    creditors={ values.creditors }
+                    featured={ values.featured }
+                    isAddDeleteDisabled={ true }
+                    owners={ values.owners }
+                    onChangeCreditors={ handleChangeCreditors }
+                    onChangeFeatured={ handleChangeFeatured }
+                    onChangeOwners={ handleChangeOwners }
+                  />
+                </Box>
 
-                  <Stack
-                    alignItems="center"
-                    columnGap={ 2 }
-                    direction="row"
-                    mt={ 5 }
+                { !!touched.owners && !!errors.owners && (
+                  <Box mt={ 0.5 } ref={ coCreatorsRef }>
+                    <ErrorMessage>{ errors.owners as string }</ErrorMessage>
+                  </Box>
+                ) }
+
+                { !!touched.creditors && !!errors.creditors && (
+                  <Box mt={ 0.5 } ref={ coCreatorsRef }>
+                    <ErrorMessage>{ errors.creditors as string }</ErrorMessage>
+                  </Box>
+                ) }
+              </Stack>
+
+              <Box py={ 5 }>
+                <HorizontalLine />
+              </Box>
+
+              <Stack columnGap={ 2 } direction="row">
+                <Button
+                  color="music"
+                  variant="secondary"
+                  width={
+                    windowWidth && windowWidth > theme.breakpoints.values.md
+                      ? "compact"
+                      : "default"
+                  }
+                  onClick={ () => navigate(-1) }
+                >
+                  Cancel
+                </Button>
+
+                { dirty && (
```

Comment:
This conditional was previously to toggle the button visibility for an update collab details or move onto the next step for minting. As minting flow is removed through this PR, does it still make sense to have this button appear only when the collab details are modified?